pandas/core/base.py

```diff
@@ -165,6 +165,12 @@ def __setattr__(self, key, value):
 class PandasDelegate(PandasObject):
     """ an abstract base class for delegating methods/properties """
 
+    @classmethod
+    def _make_accessor(cls, data):
+        raise NotImplementedError("_make_accessor should be implemented"
+                                  "by subclass.  It should return an instance"
+                                  "of `cls`.")
```

Comment:
Use instead: "_make_accessor should be implemented by subclass and return an instance of `cls`."